packages/graphql/src/components/FieldDefinition.tsx

```diff
@@ -36,45 +37,99 @@ export interface FieldDefinitionProps extends TypedBaseDeclarationProps {
  *
  * const userRef = refkey();
  *
- * <FieldDefinition
- *   name="user"
- *   type={code`${userRef}!`}
- *   description="The user who created this post"
- *   args={
- *     <>
- *       <InputValueDefinition name="id" type={code`${builtInScalars.ID}!`} />
- *       <InputValueDefinition name="includeDeleted" type={builtInScalars.Boolean} defaultValue={false} />
- *     </>
- *   }
- *   directives={
- *     <Directive
- *       name={builtInDirectives.deprecated}
- *       args={{ reason: "Use author instead" }}
+ * <>
+ *   <FieldDefinition name="id" type={code`${builtInScalars.ID}!`} />
+ *   <FieldDefinition
+ *     name="name"
+ *     type={builtInScalars.String}
+ *     description='"""User full name"""'
+ *   />
+ *   <FieldDefinition name="tags" type={code`[${builtInScalars.String}!]!`} />
```

Comment:
Seems like we should have something like `<TypeReference type={builtInScalars.String} required />` instead of these `code` blocks.

Is there any other way to reference existing types?